apps/web/lib/app-providers.tsx

```diff
@@ -54,7 +55,7 @@ const CustomI18nextProvider = (props: AppPropsWithChildren) => {
 };
 
 const AppProviders = (props: AppPropsWithChildren) => {
-  const session = trpc.useQuery(["viewer.public.session"]).data;
+  const session = proxy.public.session.useQuery().data;
```

Comment:
Example usage 

Notice that you can CMD+click `session` above and jump to the backend